browser/src/FederationPage.tsx

```diff
@@ -0,0 +1,236 @@
+    link: 'https://www.ern-rnd.eu/',
+    lead: 'Holm Graessner',
+  },
+  {
+    country: 'Germany',
+    samples: '10,000',
+    source: 'Clinical sequencing',
+    link: 'https://www.igsb.uni-bonn.de/en',
+    lead: 'Peter Krawitz',
+  },
+  {
+    country: 'Germany',
+    samples: 'TBD',
+    source: 'The German Human Genome-Phenome Archive (GHGA)',
+    lead: 'Drew Behrens',
+  },
+  {
+    country: 'Japan',
+    samples: '100,000',
+    source: 'Tomoku Medical Megabank Organization',
+    link: 'https://www.megabank.tohoku.ac.jp/english/',
+    lead: 'Soichi Ogishima',
+  },
+  {
+    country: 'Qatar',
+    samples: '25,000',
+    source: 'Qatar Biobank',
+    link: 'https://www.qphi.org.qa/',
+    lead: 'Chadi Saad',
+  },
+  {
+    country: 'Singapore',
+    samples: '10,000',
+    source: 'SG10K_Health data',
+    link: 'https://www.a-star.edu.sg/gis/our-science/precision-medicine-and-population-genomics/npm',
+    lead: 'Maxime Hebrard',
+  },
+  {
+    country: 'Taiwan',
+    samples: '1,500',
+    source: 'Taiwan Biobank',
+    link: 'https://genomes.tw/',
+    lead: 'Jacob Shujui Hsu',
+  },
+  {
+    country: 'UK',
+    samples: '50,000',
+    source:
+      'Avon Longitudinal Study of Parents and Children (ALSPC), Born in Bradford, Millennium Cohort Study (MCS), Fenland Study',
+    link: 'https://www.sanger.ac.uk/',
+    lead: 'Vivek Iyer',
+  },
+]
+
+export default () => (
+  <InfoPage>
+    <DocumentTitle title="Federated gnomAD" />
+    <PageHeading
+      // @ts-expect-error
+      id="federated-gnomad"
+    >
+      Federated gnomAD
+    </PageHeading>
+
+    <MarkdownContent dangerouslySetInnerHTML={{ __html: federationContent.html }} />
+
+    <br />
+
+    <CenteredContainer>
+      <img
+        src={FederatedGnomadMap}
+        alt="Locations of federated gnomAD data sources on a world map."
+        width="70%"
+      />
+    </CenteredContainer>
+
+    <br />
+
+    <CenteredContainer>
+      <StatsTable>
+        <thead>
+          <StatsTableHeaderRow>
+            <th>Country</th>
+            <th>Samples</th>
+            <th>Source</th>
+            <th>Lead</th>
+          </StatsTableHeaderRow>
+        </thead>
+        <StatsTableBody>
+          {datasets.map((dataset) => {
+            return (
+              <tr key={dataset.source}>
+                <td>{dataset.country}</td>
+                <td>{dataset.samples}</td>
+                <td>
+                  {dataset.link ? (
+                    // @ts-expect-error
+                    <Link href={dataset.link}>{dataset.source}</Link>
+                  ) : (
+                    dataset.source
+                  )}
+                </td>
+                <td>{dataset.lead}</td>
+              </tr>
+            )
+          })}
+        </StatsTableBody>
+      </StatsTable>
+    </CenteredContainer>
+
+    <br />
+    <br />
+
+    <CenteredContainer>
+      <LogoContainer>
+        <SingleLogo alt="" src={CanadaGnomadLogo} />
```

Comment:
Could we set these alts to "[Name of federated project] logo" or similar?

Reply:
Ah, good call, I should really leave myself `TODO` comments to clean up before submitting for a review, instead of assuming I'll remember to do the things. Will fix.